scanapi/settings.py

```diff
@@ -14,17 +18,20 @@ def __init__(self):
         super(dict, self).__init__()
 
     def save_config_file_preferences(self, config_path=None):
-        if not config_path and not self.has_default_config_file:
-            return
-
-        if not config_path:
-            user_config = load_config_file(DEFAULT_CONFIG_PATH)
-            self["config_path"] = DEFAULT_CONFIG_PATH
-            self.update(**user_config)
+        if not any(
+            [config_path, self.has_local_config_file, self.has_global_config_file]
+        ):
             return
 
-        user_config = load_config_file(config_path)
-        self.update(**user_config)
+        if config_path:
```

Comment:
Hey, can we have some tests to this part?

Reply:
Sure.

Reply:
Here we have some guidelines for testing with you want to take a look at it: https://github.com/scanapi/scanapi/blob/master/CONTRIBUTING.md#tests

Let me know if you need any help!